build.sbt

```diff
@@ -7,6 +7,18 @@ lazy val backend = (project in file("backend"))
   .dependsOn(core % "test->test;compile->compile")
   .settings(backendSettings:_*)
 
+lazy val localBackend = (project in file("local-backend"))
```

Comment:
Right but as we add supported backends, if we want to keep them under the cromwell repo umbrella it'll really blossom out. I don't know how off the top of my head but I'm pretty sure you can have sub-structure in the sbt tree to still have the individual backend jars.